[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/adr/0003-connections-to-dttp-dynamics.md

## Context

To read and modify data in DTTP, we need to be able to connect to it’s Dynamics
instance and modify data there. Dynamics has a built-in [OData API](https://www.odata.org/)
which we can use, however it exposes the data at a
low level, bypassing business logic and free of association definitions, which

> sit in the web portal app. This app is an Angular app so by it’s nature doesn’t

Reviewer: ```suggestion
sit in the web portal app. This app is an Angular app, so by its nature does not
```

#### Cons

1. Extra system to maintain with separate, more restrictive, deployment environment.

> 2. .NET isn’t core skillset of current team.

Reviewer: ```suggestion
2. .NET is not a core skillset of the current team.
```
